resources/views/our_story.blade.php

```diff
@@ -1,17 +1,17 @@
-<div class='col-md-12'>
+<div class='col-md-8 col-md-offset-2'>
```

Comment:
My bad here. Linting and refactoring led to undesired formatting changes to this.